src/md/contributing.md

```diff
@@ -10,14 +10,55 @@ This project welcomes new contributors. If you would like to help by adding new
 
 1. Create a [JIRA](https://malhar.atlassian.net/) for the work you plan to do (or assign yourself to an existing JIRA ticket)
 1. Fork the ASF github mirror (one time step):
-   https://github.com/apache/incubator-apex-core/
-1. Create a new branch from the [devel-3](https://github.com/apache/incubator-apex-core/tree/devel-3) branch. **Name your branch with the JIRA number in it, e.g. `APEX-123.my-feature`.**
+   https://github.com/apache/incubator-apex-core/  
+1. Add [incubator apex core](https://github.com/apache/incubator-apex-core) as a remote repository.  
```

Comment:
Please add one more item create local git repository. Fork, clone and adding remote repository all should mention that they are one time step.